common/utils/tf.py

```diff
@@ -36,16 +36,13 @@ def pelu(x):
         return negative + positive
 
 
-# https://stackoverflow.com/questions/39975676/how-to-implement-prelu-activation-in-tensorflow
 def prelu(_x):
-    with tf.variable_scope(_x.op.name + '_activation', initializer=tf.constant_initializer(1.0)):
-        alphas = tf.get_variable('alpha', _x.get_shape()[-1],
-                           initializer=tf.constant_initializer(0.0),
-                            dtype=tf.float32)
-        pos = tf.nn.relu(_x)
-        neg = alphas * (_x - abs(_x)) * 0.5
-
-        return pos + neg
+    """
+    :param _x: Tensor of shape [batch_size, H, W, C].
+    :return: Prelu'd tensor of shape [batch_size, H, W, C].
+    """
+    layer = tf.keras.layers.PReLU(shared_axes=[1, 2])
```

Comment:
Do we intentionally not regularize the alpha?

Reply:
I am not sure if we should. According to section 2.1 in the [PReLU paper](https://arxiv.org/pdf/1502.01852.pdf): "Even without
regularization, the learned coefficients rarely have a magnitude
larger than 1 in our experiments."